src/libs/actions/IOU.js

```diff
@@ -2815,6 +2834,11 @@ function getPayMoneyRequestParams(chatReport, iouReport, recipient, paymentMetho
                 },
             },
         },
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.NEXT_STEP}${iouReport.reportID}`,
+            value: currentNextStep,
+        },
```

Comment:
Same here, only set this onyx update object it if the current next step is not null, so we avoid queuing unnecessary updates

Reply:
For example free policy expense reports or IOU reports do not have next steps so this would always be useless